checkov/common/output/record.py

```diff
@@ -165,7 +167,8 @@ def to_string(self, compact: bool = False, use_bc_ids: bool = False) -> str:
         )
         code_lines = ""
         if self.code_block:
-            code_lines = "\n{}\n".format("".join([self._code_line_string(self.code_block)]))
+            ANSI_COLORS_DISABLED
+            code_lines = "\n{}\n".format("".join([self._code_line_string(self.code_block, not(ANSI_COLORS_DISABLED))]))
```

Comment:
@MooreDerek This is what I meant - not sure why we need the first line:
```suggestion
            code_lines = "\n{}\n".format("".join([self._code_line_string(self.code_block, not(ANSI_COLORS_DISABLED))]))
```